pymatgen/core/interface.py

```diff
@@ -2876,7 +2876,7 @@ def label_termination(slab: Structure) -> str:
 
     sp_symbol = SpacegroupAnalyzer(top_plane, symprec=0.1).get_space_group_symbol()
     form = top_plane.reduced_formula
-    return f"{form}_{sp_symbol}_{len(top_plane)}"
+    return f"{t_index+1}_{form}_{sp_symbol}_{len(top_plane)}"
```

Comment:
First I need to make clear that I'm not a maintainer (just happen to contribute to this module). Some preliminary comments on this change to `label_termination`:
- I would be on board for giving this flexibility to control the clustering tolerance here, instead of hard-coding the tolerance to be `0.25`.
-  Can we possibly update the docstring to include these two new arguments here?
- I'm sure others would strongly suggest against such breaking enhancement, especially when we could make it backwards compatible (make this prefix an empty string by default for example).

Reply:
Many thanks for the comment. I have revised the code to make backwards usage compatible by setting up a label_index bool variable. I also updated the docstring.

Reply:
No problem. Instead of adding two args for a single behavior, perhaps we could reused the single `t_index` arg, and set it to `None` by default. And add a prefix when it is not None.

Reply:
Done. `t_index + 1`was used because I prefer to label the termination to start from 1. Indeed it is not necessary.

Reply:
> Done. 

Beautiful! Thanks for the change. 

The the docstring of `label_termination` is still missing?

> `t_index + 1`was used because I prefer to label the termination to start from 1. Indeed it is not necessary.

Even you prefer the index to start from 1, adding an offset **inside** `label_termination` might still be unexpected (as you pass `index` in, and `index + 1` comes out). 

I'm not suggesting you do this here, but if you really want the index counter to start from a value other than zero, you could use the `start` arg of `enumerate(iterable, start=0)`, e.g. using `enumerate(iterable, start=1)` would start the counter from `1` instead of `0`.

Another tip being, you could [install `pre-commit` locally](https://pre-commit.com/) (`pre-commit install`) to run `pre-commit` hooks prior to each commit.